### DISCUSSION TITLE
Mute button sometimes does volume down instead; always mutes on second press when this happens

### DISCUSSION THREAD

**Discussion by kingfalcon** [MonitorControl/MonitorControl#1297] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Occasionally, the mute/unmute button behaves as volume down. This seems to happen only when the system is unmuted, and it happens sparsely. 

### Steps to reproduce

1. Hit the "mute/unmute" button on the keyboard while unmuted with sound coming through the monitor speakers
2. OSD for "reduce volume" shows up
3. Hit the button again correctly leads to "mute"

### Expected behavior

I expect the monitor sound output to mute or unmute every time I hit the keyboard button. 

### Environment Information (please complete the following information)

```markdown
- macOS version: Monterey 12.6.2 
- Mac model: MacBook Pro (13-inch, 2020, Four Thunderbolt 3 ports)
- MonitorControl version: 4.1.0 Build 7034
- Monitor(s): Dell U3419W
- Apple Silicon/M1 (yes or no): No

Keyboard: Keychron C1
```

**Comment by waydabber** [comment]
This seems to be a monitor issue. You might want to play with some of the mute related settings of BetterDisplay to see if there is any combination of settings that work with that display.

https://github.com/waydabber/BetterDisplay/issues/1248
